src/plugins/home/public/services/tutorials/tutorial_service.ts

```diff
@@ -74,6 +79,14 @@ export class TutorialService {
         }
         this.tutorialModuleNotices[id] = component;
       },
+
+      registerCustomStatusCheck: (name: string, fnCallback: CustomStatusCheckCallback) => {
```

Comment:
I know the tests on this file aren't very exhaustive and this doesn't do very much, but at least some basic coverage that this works needs to be added. Especially since this plugin is abandonware & touched by many teams, I think it's important we don't break this by accident.

Reply:
Tests added https://github.com/elastic/kibana/pull/100816/commits/496f71316bc331264f81b9423a3e82ea933ac4d2